web-services/deploy/docs/docs/query_help.html

```diff
@@ -258,5 +258,26 @@ <h3>Excerpts Function</h3>
 		<li>excerpt.fields="field/offset"</li>
 		<li>excerpt.fields="field1/offset,field2/offset,..."</li>
 	</ul>
+	<h3>Language Expansion</h3>
+	<p class="left">Datawave provides limited support for Lucene-based language expansion. A user may specify a set of language short codes that determine which
+		analyzers are applied to a query.</p>
+	<p class="left">Language analyzers may support stemming, lemmatization, unigramming and bigramming. All options are enabled by default, but Lucene only
+		supports certain features by default for certain languages. Stemming is supported for most languages, and bigrams are supported or CJK languages.</p>
```

Comment:
```suggestion
		supports certain features by default for certain languages. Stemming is supported for most languages, and bigrams are supported for CJK languages.</p>
```
Seems like this should be 'for'?